### PR TITLE
Added built jar from private ingest sdk

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
We will replace it with beta version of ingest SDK once it is released. 
So this is just a placeholder PR. 

PS:
- Also update `pom_confluent.xml`
- Also note base branch, it is kept as `japatel-279791-master-streaming-snowpipe-prpr` instead of master

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,6 +68,12 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+
+<!--        Placeholder for local import of an unpublished library to maven repo. (i.e this is a local repository)-->
```
Comment (sfc-gh-tjones) [inline pom.xml:72]: Nit, is the formatting correct here?

**Review by sfc-gh-tjones — APPROVED** [review]
This looks fine as a placeholder PR. Thanks!

**Review by sfc-gh-mnaides — APPROVED** [review]

**Comment by sfc-gh-tzhang** [comment]
What's the purpose of this placeholder PR? I will give a review once we have the released sdk, thanks!

**Comment by sfc-gh-japatel** [comment]
> What's the purpose of this placeholder PR? I will give a review once we have the released sdk, thanks!

1. We are pushing this into a master branch of streaming ingest. japatel-279791-master-streaming-snowpipe-prpr
2. Adding this as a placeholder helps us to see if we break any existing code path. (Tests on Github actions) (Except GCS which is flaky and needs multiple runs)

File: pom.xml (hunk 1 above)
Comment (sfc-gh-japatel) [reply pom.xml:72]: Will remove this once we have beta version of ingest sdk with streaming API released.

**Pull request merged**